ocw/lib/ec2.py

```diff
@@ -70,7 +77,7 @@ def cleanup_snapshots(self, valid_period_days: float) -> None:
             response = self.ec2_client(region).describe_snapshots(OwnerIds=['self'])
             self.log_dbg(f"Found {len(response['Snapshots'])} snapshots in {region}")
             for snapshot in response['Snapshots']:
-                if EC2.is_outdated(snapshot['StartTime'], valid_period_days):
+                if self.is_outdated(snapshot['StartTime'], valid_period_days) or self.is_cancelled(snapshot):
```

Comment:
is_outdated is static method , while what you have written will work I prefer to have `EC2.is_outdated` this way we underline that we calling method which has nothing to do with certain object instance and does not operate any runtime data